tests/acceptance/acceptancesuites/facetssuite.js

```diff
@@ -0,0 +1,73 @@
+import {
+  FACETS_PAGE,
+  VERTICAL_SEARCH_URL_REGEX
+} from '../constants';
+import FacetsPage from '../pageobjects/facetspage';
+import { MockedVerticalSearchRequest } from '../fixtures/responses/vertical/search';
+import {
+  registerIE11NoCacheHook
+} from '../utils';
+import SearchRequestLogger from '../searchrequestlogger';
+
+fixture`Facets page`
```

Comment:
If possible, I think we should split these tests up as well. Is there a cost associated with that? We don't set up the page for each test right, we just run the `beforeEach` hook?

Reply:
The test currently goes
1. select a facet
2. reset the facets
3. select 3 facets
4. reset the facets

I think we can cut this test in half in after the first reset. I think I might have been worried in the past about the SDK being able to select facets the first time but not the second time, but realistically speaking that shouldn't be an issue.